src/pages/[...wordpressNode].js

```diff
@@ -5,10 +5,11 @@ export default function Page(props) {
 }
 
 export async function getStaticProps(ctx) {
-  // use isr for the shop and product pages
+  // use isr for the shop, search and product pages
```

Comment:
Do we really need this comment?

![Fry Squint](https://media.giphy.com/media/cFgb5p5e1My3K/giphy.gif)

Reply:
Nah prob not! Removed